content/src/ports/activeEntities.ts

```diff
@@ -0,0 +1,175 @@
+import { Deployment, Entity, EntityId, Pointer } from 'dcl-catalyst-commons'
+import LRU from 'lru-cache'
+import { EnvironmentConfig } from '../Environment'
+import { getDeployments } from '../service/deployments/deployments'
+import { AppComponents } from '../types'
+
+export type ActiveEntities = {
+  withPointers(...pointers: Pointer[]): Promise<Entity[]>
+  withIds(...entityIds: EntityId[]): Promise<Entity[]>
+  inactivate(...pointers: Pointer[]): void
+  activate(pointer: Pointer, entity: Entity): void
+}
+
+const mapDeploymentsToEntities = (deployments: Deployment[]): Entity[] => {
```

Comment:
This method is only for the content field? :(

Reply:
yes :(